src/promise_provider.ts

```diff
@@ -14,6 +14,7 @@ const store: PromiseStore = {
  */
 export class PromiseProvider {
   /** Validates the passed in promise library */
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
why is this needed? I just checked in vscode and there were no errors in this file

Reply:
I was able to get rid of this but you should see an error for any function that uses `any` that's exported from a module.